common

```diff
@@ -110,6 +110,20 @@ printline() {
 }
 
 
+get_worlds_dir() {
+    worlds_dir="/config/worlds_local"
+    if [ ! -d "${WORLDS_DIR}" ]; then
```

Comment:
Where is (capital) `$WORLDS_DIR` coming from at this point? From a global context or should it be the lower case one defined in the line above?